assets/js/apicalls.js

```diff
@@ -0,0 +1,82 @@
+
+
+const omdbAPIkey = "5af38ba4";
+var test = document.getElementById('root');
+
+function fetchAllGhibliMovies() {
+  apicall();
```

Comment:
These seem to be placeholders? They can be removed, since apicalls.js will be shared by both webpages.

If you need code from these functions to render index.html, they should be put in script.js.